x/data/go.mod

```diff
@@ -9,8 +9,7 @@ require (
 	github.com/gorilla/mux v1.8.0
```

Comment:
Trying to figure out how to fix that:
```go mod tidy
go: finding module for package github.com/regen-network/regen-ledger/types/testutil/network
go: finding module for package github.com/regen-network/regen-ledger/types/testutil
go: found github.com/regen-network/regen-ledger/types in github.com/regen-network/regen-ledger/types v0.0.0-00010101000000-000000000000
go: found github.com/regen-network/regen-ledger/types/module/client/cli in github.com/regen-network/regen-ledger/types v0.0.0-00010101000000-000000000000
go: found github.com/regen-network/regen-ledger/types/module/server in github.com/regen-network/regen-ledger/types v0.0.0-00010101000000-000000000000
go: found github.com/regen-network/regen-ledger/types/module in github.com/regen-network/regen-ledger/types v0.0.0-00010101000000-000000000000
go: github.com/regen-network/regen-ledger/types@v0.0.0-00010101000000-000000000000 requires
        github.com/regen-network/regen-ledger/app@v0.0.0-00010101000000-000000000000 requires
        github.com/regen-network/regen-ledger/x/data@v0.0.0-00010101000000-000000000000: invalid version: unknown revision 000000000000
```

Reply:
`github.com/regen-network/regen-ledger/types@v0.0.0-00010101000000-000000000000 requires
        github.com/regen-network/regen-ledger/app@v0.0.0-00010101000000-000000000000` I think here's the problem, and can be solved by https://github.com/regen-network/regen-ledger/pull/322#discussion_r612573799